packages/flutter/lib/src/material/bottom_sheet_theme.dart

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+/// Defines default property values for [BottomSheet] widgets.
+///
+/// Descendant widgets obtain the current [BottomSheetThemeData] object
+/// using `Theme.of(context).bottomSheetTheme`. Instances of
+/// [BottomSheetThemeData] can be customized with
+/// [BottomSheetThemeData.copyWith].
+///
+/// Typically a [BottomSheetThemeData] is specified as part of the
+/// overall [Theme] with [ThemeData.bottomSheetTheme].
+///
+/// All [BottomSheetThemeData] properties are `null` by default.
+/// When null, the [BottomSheet] will provide its own defaults.
+///
+/// See also:
+///
+///  * [ThemeData], which describes the overall theme information for the
+///    application.
+class BottomSheetThemeData extends Diagnosticable {
+  /// Creates a theme that can be used for [ThemeData.bottomSheetTheme].
+  const BottomSheetThemeData({
+    this.backgroundColor,
+    this.elevation,
+    this.shape,
+  });
+
+  /// Color to be used for the [BottomSheet]'s background.
```

Comment:
Is this actually the color of the bottom sheet's material?

Reply:
Yes, modified the documentation to match other themes